lib/src/working_copy.rs

```diff
@@ -307,6 +307,12 @@ pub enum SnapshotError {
     InternalBackendError(#[from] BackendError),
     #[error(transparent)]
     TreeStateError(#[from] TreeStateError),
+    #[error("New file {path} containing {size} exceeds configured limit {max_size}")]
```

Comment:
And printing human-readable 1MiB here too.

Also probably a hint should be printed that you need to od one of 3 things, either track the file explicitly, add it to gitignore or up the limit.

(looking at a comment from yuja below those 3 things should indeed prevent this error from happening heh)

wait, there seems to be no `jj track` (the reverse of `jj untrack`), hmmm..

Reply:
Yeah, we don't have an explicit tracking yet. Because the new check only triggers for new files, it should be viable to raise/remove the limit temporarily, snapshot, and then restore it to default (if desired), so I don't think that needs to be a blocker, though it would be nice to have.

Reply:
Implemented pretty-printing of sizes.